www/config.yml

```diff
@@ -489,6 +489,10 @@ chapters:
   pro_network: Python Software Foundation Meetup Pro Network
   twitter: PyLadiesMEL
   website: http://melbourne.pyladies.com/
+- email: Mohali@pyladies.com
+  image: mapmaker.png
```

Comment:
Does this image need to be uploaded/included in this PR?

Reply:
@Mariatta image is already uploaded in  www/_assets/images/ directory.

Reply:
The image in the www/_assets/images directory is `mapmarker` instead of `mapmaker`

https://github.com/pyladies/pyladies/blob/master/www/_assets/images/mapmarker.png